src/main/java/frc/robot/Constants.java

```diff
@@ -32,13 +32,11 @@
         public static final double MAXIMAL_ACCELERATION = 0; // [ticks/100ms*sec]
         public static final boolean VOLTAGE_COMPENSATION = true;
 
-        public static final double MAX_VELOCITY = Math.PI * 2 / 3; //[rad/s]
+        public static final double MAX_VELOCITY = Math.PI * 2 / 3; //[rad// /s]
```

Comment:
Why? `rad/s` is good 🙂.

Reply:
I believe it was mistyped and not on purpose